rc/lsp.kak

```diff
@@ -128,6 +128,11 @@ info=$lsp_info \
             lines++
         }
 
+        if (ENVIRON["kak_opt_lsp_modeline_message_requests"]) {
```

Comment:
`kak_opt_lsp_modeline_message_requests` is not exported at this point, need to add it to this line:

```
    content=$(eval "${kak_opt_lsp_show_hover_format}") # kak_opt_lsp_hover_max_lines kak_opt_lsp_modeline_code_actions
```